specs/code-lists/CSV/cost_type.csv

```diff
@@ -0,0 +1,5 @@
+Code,Label,Definition
+Structure,Structure,A physical structure such as a building or piece of infrastructure.
+Content,Content,Physical contents of a structure.
+Product,Product,A product produced by a business that may or may not be physical items.
+Disruption,Disruption (Business Interruption),Disruption to the planned operations of a business.
```

Comment:
I think the codes in this list should better explain the relationship between the cost and the loss of the exposed assets.

```suggestion
Structure,Structure,Costs associated with the loss of a physical structure such as a building or piece of infrastructure.
Content,Content,Costs associated with the loss of the physical contents of a structure.
Product,Product,Costs associated with the loss of products produced by businesses, which may or may not be physical items.
Disruption,Disruption (Business Interruption),Costs associated with disruption to the planned operations of a business.
```

I think that we should check that this codelist is sufficient to describe the costs associated with the assets in `exposure_category.csv`. For example, I'm not sure which of these codes would apply to the costs associated with the loss of crops in the event of a farm being flooded.

Reply:
yeah this was one I was going to run past you and then run past Stu before merging into dev, I just figured the conversation about the descriptions could happen in the PR